addons/udes_stock/models/stock_move.py

```diff
@@ -41,7 +41,6 @@ def _prepare_info(self):
                 "quantity_done": self.quantity_done,
                 "product_id": self.product_id.get_info()[0],
                 "moves_line_ids": self.move_line_ids.get_info(),
-                "product_packaging": self.product_packaging.name,
```

Comment:
This appears to undo a change in an earlier commit from this series: is this deliberate?